File: app/src/main/java/com/rikkimikki/torrentino/domain/pojo/search/TopResult.java
package com.rikkimikki.torrentino.domain.pojo.search;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class TopResult {
    @SerializedName("global")
    @Expose
    private Movie__1 global;
    @SerializedName("__typename")
    @Expose
    private String typename;

    public Movie__1 getGlobal() {
        return global;
    }

    public void setGlobal(Movie__1 global) {
        this.global = global;
    }

    public String getTypename() {
        return typename;
    }

    public void setTypename(String typename) {
        this.typename = typename;
    }
}

File: app/src/main/java/com/rikkimikki/torrentino/domain/pojo/search/TopResult.kt
package com.rikkimikki.torrentino.domain.pojo.search

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class TopResult (
    @SerializedName("global")
    @Expose
    val global: Movie__1,

    @SerializedName("__typename")
    @Expose
    val typename: String
)